src/test/java/core/basesyntax/db/StorageDaoImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.db;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.models.Fruit;
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageDaoImplTest {
+    private static StorageDao storage;
+    private static Fruit testFruit1;
+
+    @BeforeClass
+    public static void beforeClass() {
+        storage = new StorageDaoImpl();
+    }
+
+    @Test
+    public void add_FruitIntegerPair_Ok() {
```

Comment:
давай дотримуватись однокового стилю в назвах методів.
```suggestion
    public void add_newFruitQantity_Ok() {
```